src/app/static/src/tests/components/runReport/runReport.test.ts

```diff
@@ -32,7 +38,7 @@ describe("runReport", () => {
         {name: "report2", date: null}
     ];
 
-    const getWrapper = (reports, propsData = props) => {
+    const getWrapper = (report = reports, propsData = props) => {
```

Comment:
This doesn't look right - the next line expects a variable called `reports`, not `report`